### PR TITLE
Simple client with geolocation

### PR DESCRIPTION
**Pull request by KrivichaninDenis**
**Create simple client**
This version does not display the geolocation on a mobile phone
[demo](https://pathetic-partner.surge.sh/)

When call the function  [_geoFindMe_](https://github.com/pub-t/plain-destination/blob/simple-geolocation/js/script.js#L9)
**error** callback work succesfully.
**success** callback did't work

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -1,11 +1,53 @@
...
+    document.getElementById('fromBlock').classList.add('animated','bounceInLeft');
+    document.getElementById('fromBlock').style.display = 'none';
+
+    geoFindMe();
+});
+
+function geoFindMe() {
+    var output = document.getElementById("out");
+    var el = document.getElementById('fromBlock');
+    var timeoutVal = 10 * 1000 * 1000;
```
Comment (irnc) [inline js/script.js]: In which units this value is set? Timer is essential, please read http://stackoverflow.com/a/3885172/1065806

Comment (KrivichaninDenis) [inline js/script.js]: Problem resolved. Problem was in the phone. Positioning was only by _GPS_

Comment (irnc) [inline js/script.js]: Please document this issue, as real users would face the same problem too, and we need to workaround it and guide users, so they are not confused by non-functioning app.

**Comment by KrivichaninDenis** [comment]
Add eslint + request

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "bus-client",
+  "version": "0.0.1",
+  "description": "<h1><b>Plain-destination</b></h1>\r <h2>Structure</h2>\r | filename  | Content a file |\r | ------------- | ------------- |\r | index.html  | Main file in page-proofs  |\r | css/style.css  | Main styles  |\r | css/reset.css  | Reset CSS of Eric Mayer |\r | css/fonts.css  | Connect different the fonts |\r | css/media.css  | Main styles  |\r | js/script.css  | Work for element of form |",
```
Comment (irnc) [inline package.json]: I guess we should have different description ;)

File: package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  "bugs": {
+    "url": "https://github.com/pub-t/plain-destination/issues"
+  },
+  "homepage": "https://github.com/pub-t/plain-destination#readme",
+  "devDependencies": {
+    "eslint": "^2.12.0",
+    "eslint-config-airbnb": "^9.0.1",
+    "eslint-plugin-import": "^1.8.1",
+    "eslint-plugin-jsx-a11y": "^1.4.2",
+    "eslint-plugin-react": "^5.1.1"
```
Comment (irnc) [inline package.json]: This isn't React app, we should not add it's plugin.

File: js/script.js [hunk 4]
```diff
@@ -1,11 +1,93 @@
...
+  } else {
+    if (toBlock.length !== 0) {
+      $('#submitButton').removeAttr('disabled');
+    } else {
+      $('#submitButton').attr('disabled', 'disabled');
+    }
+  }
+
+  // document.getElementById('submitButton')
+  //     .disabled = fromBlock.value || toBlock.value ? false : 'disabled';
```
Comment (irnc) [inline js/script.js]: Commented out code should not be committed to repository.

**Comment by KrivichaninDenis** [comment]
I will correct these errors soon

**Pull request merged**